game/unitmap.py

```diff
@@ -43,6 +43,11 @@ class Building:
     ground_object: BuildingGroundObject
 
 
+@dataclass(frozen=True)
+class Scenery:
+    ground_object: SceneryGroundObject
```

Comment:
Just use `Building` and `add_building`. They're functionally equivalent in the debriefing, and there's no point displaying them differently to the player. They killed a building, they don't care how it was implemented.

Reply:
I've tried this a few different ways now and haven't been successful.  Maybe someone else will have better luck than me.

Reply:
Pushed a fix. Have a look.